src/EpiEasyEvents.Tests/EpiEasyEvents.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
I'm not sure here. Maybe since "main" project is targetting .net 6 we should keep tests aligned with it and target .net 6 as well?